databricks_asset_bundles/04.deploy_model.py

```
@@ -0,0 +1,54 @@
+parser.add_argument(
+    "--root_path",
+    action="store",
+    default=None,
+    type=str,
+    required=True,
+)
+
+args = parser.parse_args()
+root_path = args.root_path
+
+config_path = (f"project_config.yml")
```

Ruff (0.7.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)